changelog/unreleased/enhancement-move-share-indicators

```diff
@@ -1,9 +1,11 @@
 Enhancement: Move share indicators
 
 We've moved the share/status indicators into a separate column
-and adjusted the design in ODS.
+and adjusted the design in ODS. We also removed links from
+shared-with-me-page.
```

Comment:
Shared with others - but please create a separate changelog items for this enhancement (the removal of links from shared with others).